client/src/search.tsx

```diff
@@ -223,10 +223,7 @@ function InnerSearchNavigateWidget(props: InnerSearchNavigateWidgetProps) {
       return [];
     }
 
-    // The iPhone X series is 812px high.
-    // If the window isn't very high, show fewer matches so that the
-    // overlaying search results don't trigger a scroll.
-    const limit = window.innerHeight < 850 ? 5 : 10;
+    const limit = 10;
```

Comment:
Maybe it does make sense to keep the old behavior for windows of less than 850px, since nobody complained about it?

Reply:
Having only 5 items actually looks quite neat on mobile.